test/expect/TestScript.test_optional_scalartype.expect

```diff
@@ -0,0 +1,13 @@
+graph(%a : Tensor,
+      %b : Tensor):
```

Comment:
Could you find another way to test this other than expect files ? We do not use them. Consider using our filecheck tool as a way of comparing expected textual output

Reply:
removed the expect test and added FileCheck tests